Rhinoceros_Adapter/AdapterActions/Push.cs

```diff
@@ -0,0 +1,62 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using BH.oM.Data.Requests;
+using BH.oM.Adapter;
+using BH.oM.Base;
+using System.Reflection;
+
+namespace BH.Adapter.Rhinoceros
+{
+    public partial class RhinocerosAdapter : BHoMAdapter
+    {
+        public override List<object> Push(IEnumerable<object> objects, String tag = "", PushType pushType = PushType.AdapterDefault, ActionConfig actionConfig = null)
```

Comment:
Not sure we really should be overriding this? Or, might be me missing the need?

Reply:
Good questions. I am just following a pattern of override use on Push and Pull observed in other Adapters. I can't answer why though!

Reply:
ok, think those cases should be seen as exceptions in general! If it works without overriding, I think we should skip overriding! :)

Reply:
I cannot find an adapter where Push and Pull are not overridden! Maybe @alelom can clarify why this is set up in this way?

Reply:
Non of the structural adapters override it, or at least, any override is only made for the sake of doing some initial checks on the adapter or so, before the `base.Push()` is called :)